mythril/laser/ethereum/instructions.py

```diff
@@ -717,17 +717,15 @@ def calldatacopy_(self, global_state: GlobalState) -> List[GlobalState]:
             log.debug("Unsupported symbolic memory offset in CALLDATACOPY")
             return [global_state]
 
-        dstart_sym = False
         try:
-            dstart = util.get_concrete_int(op1)
+            dstart = util.get_concrete_int(op1)  # type: Union[int, BitVec]
```

Comment:
Should this just be `int`? Also on line 728? Then 727 is also unneeded.

Reply:
That meant the value of fstart can be either int if typeerror isn’t raised, else if type error is raised it will be bitvec right?